client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -140,6 +140,7 @@ export default function SitesOverview(): ReactElement {
 							</div>
 						</div>
 						<SectionNav
+							applyUpdatedStyles
```

Comment:
I'm curious if we can use a better name for this property because `applyUpdatedStyles` isn't bringing any context to me.

What do you think about using `isJetpackCloud` or something like that?

Reply:
We have a new design for tabs which we cannot update throughout Jetpack Cloud as some tiny changes might be needed at the individual component levels, so don't think we can name it `isJetpackCloud` yet.